monai/deploy/operators/dicom_series_to_volume_operator.py

```diff
@@ -90,6 +95,35 @@ def generate_voxel_data(self, series):
         vol_data = np.stack([s.get_pixel_array() for s in slices], axis=0)
         vol_data = vol_data.astype(np.int16)
```

Comment:
In a recent project I used the following code in a subclass of DICOMSeriesToVolumeOperator in  generate_voxel_data() in order to apply the stored windowing (with good results):

`from pydicom.pixel_data_handlers.util import apply_voi_lut`

```
slices = series.get_sop_instances()
# The sop_instance get_pixel_array() returns a 2D NumPy array with index order
# of `HW`. The pixel array of all instances will be stacked along the first axis,
# so the final 3D NumPy array will have index order of [DHW]. This is consistent
# with the NumPy array returned from the ITK GetArrayViewFromImage on the image
# loaded from the same DICOM series.
mod_slices = []
for s in slices:
    mod_s = apply_voi_lut(s.get_native_sop_instance().pixel_array, s.get_native_sop_instance())
    mod_slices.append(mod_s)
vol_data = np.stack(mod_slices, axis=0)

vol_data = vol_data.astype(np.int16)
```

Reply:
Should I contribute this in a separate PR?

Reply:
I think it is worth its own issue and corresponding PR, thanks.